CHANGELOG.md

```diff
@@ -163,7 +163,7 @@
 - PR #2557 fix cudautils import in string.py
 - PR #2521 Fix casting datetimes from/to the same resolution
 - PR #2560 Remove duplicate `dlpack` definition in conda recipe
-- PR #2567 Fix ColumnVector.fromScalar issues while dealing with null scalars
+- PR #2565 Orc reader: fix incorrect data decoding of int64 data types
```

Comment:
Please don't delete lines for other PRs!

Reply:
Good catch @harrism, resolved!